test/integration/mysql-public/mysql_public_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// define test package name
+package mysql_public
+
+import (
+	"fmt"
+	"testing"
+
+	// import the new Blueprints test framework modules for testing and assertions
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/gcloud"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test/pkg/tft"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMySqlPublicModule(t *testing.T) { // name the function as Test*
+
+	// define constants for all required assertions in the test case
+	const databaseVersion = `MYSQL_5_6`
+	const gceZone = `us-central1-c`
+	const region = `us-central1`
+	const tier = `db-n1-standard-1`
+	const authNetName = `sample-gcp-health-checkers-range`
+	const authNetCidr = `130.211.0.0/28`
+	const dbFlagName = `log_bin_trust_function_creators`
+	const dbFlagValue = `on`
+
+	mySqlT := tft.NewTFBlueprintTest(t)                   // initialize Terraform test from the Blueprints test framework
+	mySqlT.DefineVerify(func(assert *assert.Assertions) { // define and write a custom verifier for this test case
```

Comment:
suggest we move comments above line for readability throughout
```suggestion
        // define and write a custom verifier for this test case
	mySqlT.DefineVerify(func(assert *assert.Assertions) { // define and write a custom verifier for this test case
```

Reply:
Acked